libs/app-mtp/src/lib/dotnet-mtp/utils/handle-command.ts

```diff
@@ -25,7 +26,8 @@ const PREFIX_LOG = `[app-mtp/dotnet-mtp]`
 export async function runCommand(
```

Comment:
Since there's no `this` required inside this function's body, I'd stick to one convention and use the arrow function instead.